tests/test_json.py

```diff
@@ -129,6 +129,20 @@ def test_json():
     assert records == new_records
 
 
+def test_boolean_conversion():
+    schema = {
+        "type": "record",
+        "name": "test_boolean_conversion",
+        "namespace": "test",
+        "fields": [
+            {"name": "boolean", "type": "boolean"},
+        ],
+    }
+    records = [{"boolean": 1}, {"boolean": 10}, {"boolean": 0}]
+    converted_records = [{"boolean": True}, {"boolean": True}, {"boolean": False}]
+    assert converted_records == roundtrip(schema, records)
+
+
```

Comment:
Although this shows the type conversion, it does not show what the serialised representation is and how the other side reads it though. If it is JSON, it needs to be true/false written. If it is binary, it needs to be a single byte 0/1 written according to the spec. The other end needs a consistent read as well. One would expect the Avro union data types work properly too .

Reply:
The change was made for `AvroJSONEncoder`. Presumably it doesn't affect binary serialisations?